src/protagonist/API/Features/Image/Requests/CreateOrUpdateImage.cs

```diff
@@ -99,21 +99,20 @@ public async Task<ModifyEntityResult<Asset>> Handle(CreateOrUpdateImage request,
             request.AssetMustExist,
             request.AlwaysReingest,
             false,
-            async updatedAsset =>
-            {
-                if (updatedAsset.Family == AssetFamily.Timebased)
-                {
-                    // Timebased asset - create a Batch record in DB and populate Batch property in Asset
-                    await batchRepository.CreateBatch(updatedAsset.Customer, updatedAsset.AsList(), cancellationToken);
-                }
-            },
-            cancellationToken
+            cancellationToken: cancellationToken
         );
         
         var modifyEntityResult = processAssetResult.Result;
 
         if (modifyEntityResult.IsSuccess)
         {
+            if (modifyEntityResult.Entity!.HasDeliveryChannel(AssetDeliveryChannels.Timebased))
```

Comment:
Should this be `if (updatedAsset.Family == AssetFamily.Timebased)` to keep consistent with previous logic?